source/features/add-tag-to-commits.tsx

```diff
@@ -137,7 +140,10 @@ async function init(): Promise<void | false> {
 			// There was no tags for this commit, save that info to the cache
 			commitsWithNoTags.push(targetCommit);
 		} else if (targetTags.length > 0) {
-			select('.commit-meta', commit)!.append(
+			select([
+				'.commit-meta', // Pre "Repository refresh" layout
+				'p + div.d-flex'
```

Comment:
```suggestion
p + div.d-flex:not(.Details-content--hidden)
```

If someone wants to send a pr. If not i can later tonight